lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -303,6 +305,19 @@ public void clearComposition() {
 
 // <editor-fold desc="animator">
 
+  @Override public void start() {
+    playAnimation();
+  }
+
+  @Override public void stop() {
+    lazyCompositionTasks.clear();
```

Comment:
@alexjlockwood If you call stop then play, does AVD resume from where stop was called or restart from 0?

Reply:
Calling `stop()` then `start()` on an AVD will restart the animation completely (it doesn't support pausing/resuming).

AVD source code: https://github.com/aosp-mirror/platform_frameworks_base/blob/oreo-mr1-release/graphics/java/android/graphics/drawable/AnimatedVectorDrawable.java#L893-L945

Reply:
The `Animatable` docs don't specify how this method should be implemented exactly... so it is probably fine to do it however you want (i.e. `stop()` could call `cancelAnimation()` or `pauseAnimation()` or something).

Reply:
@alexjlockwood let's mimic AVD. Just call cancelAnimation

Reply:
We can do that, but that doesn't exactly mimic AVD. :)

As you can see from the [source code](https://github.com/aosp-mirror/platform_frameworks_base/blob/oreo-mr1-release/graphics/java/android/graphics/drawable/AnimatedVectorDrawable.java#L893-L945), AVD implements `Animatable#stop()` by calling `animator.end()`, not `animator.cancel()`.

Reply:
@alexjlockwood Good point. I've never directly exposed end. Maybe we should do that. What do you think?

Reply:
You mean by creating a `stopAnimation()` on `LottieDrawable` and `LottieValueAnimator` (and then have the `LottieValueAnimator` call its own `end()` method)?

Reply:
@alexjlockwood Expose `endAnimation` that proxies to `end()` then have `stop()` call `endAnimation()`
What do you think?

Reply:
Updated the pull request with your suggestion. LMK what you think!